src/ucp/wireup/wireup.c

```diff
@@ -280,6 +280,10 @@ static double ucp_runtime_score_func(ucp_worker_h worker, uct_iface_attr_t *ifac
                  UCT_IFACE_FLAG_ATOMIC_SWAP64 | UCT_IFACE_FLAG_ATOMIC_CSWAP64;
     }
 
+    if (context->config.features & UCP_FEATURE_WAKEUP) {
+        flags |= UCT_IFACE_FLAG_WAKEUP;
```

Comment:
@yosefe matter of configuration... we need to say what's the appropriate default, and what should happen with unsupported transports. I think it's reasonable to exclude transports which do not explicitly support this...